pkg/standalone-utils/contracts/relayer/AaveWrapping.sol

```diff
@@ -47,13 +47,14 @@ abstract contract AaveWrapping is IBaseRelayerLibrary {
         IERC20 dynamicToken = fromUnderlying ? staticToken.ASSET() : staticToken.ATOKEN();
 
         // The wrap caller is the implicit sender of tokens, so if the goal is for the tokens
-        // to be sourced from outside the relayer, we must first them pull them here.
+        // to be sourced from outside the relayer, we must first pull them here.
         if (sender != address(this)) {
             require(sender == msg.sender, "Incorrect sender");
             _pullToken(sender, dynamicToken, amount);
         }
 
         dynamicToken.approve(address(staticToken), amount);
+        // Use 0 for the referral code
```

Comment:
Do we want to set one of these up maybe?

Reply:
I was wondering about that. Is it worth passing one in at construction time? I could add that

Reply:
Aave has confirmed that they are not and will not use these referral codes, so we don't need to do anything.